src/tests/ftest/recovery/start_subcommand.py

```diff
@@ -0,0 +1,158 @@
+"""
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from general_utils import check_file_exists
+from recovery_test_base import RecoveryTestBase
+from run_utils import run_remote
+
+
+class StartSubcommandTest(RecoveryTestBase):
+    """Test dmg check start subcommands.
+
+    Subcommand list. (Not all subcommands are tested here.)
+    -n, --dry-run           Scan only; do not initiate repairs.
+    -r, --reset             Reset the system check state.
+    -f, --failout=[on|off]  Stop on failure.
+    -a, --auto=[on|off]     Attempt to automatically repair problems.
+    -O, --find-orphans      Find orphaned pools.
+    -p, --policies=         Set repair policies.
+
+    :avocado: recursive
+    """
+
+    def test_check_start_reset(self):
+        """Test dmg check start --reset subcommand.
+
+        See the state diagram attached to the ticket.
+
+        1. Create a pool.
+        2. Inject orphan pool fault.
+        3. Start the checker with interactive mode. This will move the state to "checking"
+        from "unchecked".
+        4. Verify that the orphan pool is detected.
+        5. Stop the checker. The state is now at "stopped".
+        6. Remove the pool directory from the mount point.
+        7. Start the checker without --reset. State is back to "checking".
+        8. Verify that the action entry is still there.
+        9. Stop the checker. State is "stopped".
+        10. Start the checker with --reset. The state should have transitioned to
+        "unchecked", then "checking".
+        11. Verify that the action entry is empty and the status is COMPLETED.
+
+        Jira ID: DAOS-17623
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=recovery,cat_recov
+        :avocado: tags=StartSubcommandTest,test_reset
```

Comment:
Because the name was changed
```suggestion
        :avocado: tags=all,full_regression
         :avocado: tags=vm
         :avocado: tags=recovery,cat_recov
         :avocado: tags=StartSubcommandTest,test_check_start_reset
```